.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,29 +9,23 @@ Thank you for making a submission to the *aws-doc-sdk-examples* repository. For
 The submitter has:
 
 - [ ] Added the default copyright notice to all files.
```

Comment:
@brmur  Perhaps we should include the text of Amazon's default license.  

There is an [Amazon wiki](https://w.amazon.com/bin/view/Open_Source/LicensingForGitHubProjects/Apache-2.0).  

I think the source file requires an Amazon copyright, not the submitter's name on the copyright.  Perhaps adding the copyright should be handled by the Amazon reviewer.  So that the submitter is not confused.

Reply:
Suggested edits: 
Add the following copyright notice to the end of each file: _Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved._  This applies copywrite as described in this repo's [LICENSE agreement](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/LICENSE).

Reply:
Actually, I think this does not apply copyright.  The copyright exists already and is defined by the license.  The sentence is more of a notification.  This text is out of date.  Also, since it is a checkbox, I have retained the past participle.  Checked that "added".